### ISSUE TITLE
Inconsistencies on long code reviews

### ISSUE THREAD

**Issue by paulodamaso** [zerocracy/farm#1751]
0crat is showing a message when architect merge code reviews who took too much time to be merged:
```
Code review was too long (n days), architects (@paulodamaso) were penalized, see §55
```

But there are some problems:
1) no penalty was given (I received the same 10 points as any other merge);
2) 0crat should calculate the time of the code review based in another factors besides PR date: in this case I've became the architect of the project and made the merge in one or two days, but the PR was late due previoues `ARC` inactivity. 0crat should reset this time when `ARC` is changed, so the new arqchitect is not penalized.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by g4s8** [comment]
@paulodamaso we have some options about second issue: you can be added as a second architect to a project **or** first arc may resign the role first and then assign to you as a single architect. I think we should do nothing in first case, but handle only second scenario, but I can't figure out how to do it correctly, because "reset the time" (as you suggested) is not the correct way to do it, because it affects performer metrics, not anly architects (e.g. speed). @yegor256 maybe you can suggest something here?

**Comment by paulodamaso** [comment]
@g4s8 Updating the issues, 1) is happening correctly, checked in awards and it shows the penalty og 60 points due to delayed reviews.

**Comment by paulodamaso** [comment]
@g4s8 please take a look

**Comment by g4s8** [comment]
@yegor256 please look here, maybe we can skip penalty for new ARC? E.g. do not send negarive points for first 2 days, since role assigned.

**Comment by yegor256** [comment]
@g4s8 yes, I like the idea of "quarantine" period for a new role. Two days look reasonable.

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by paulodamaso** [comment]
@g4s8 I received the points but not the cash for this one, please take a look

**Comment by g4s8** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/1751#issuecomment-451184432))

@g4s8 This is what I know about this job in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#1751](https://github.com/zerocracy/farm/issues/1751) is [in scope](http://www.zerocracy.com/policy.html#14) for 9days
  * The role is `DEV`
  * The job is not assigned to anyone
  * The [budget](http://www.zerocracy.com/policy.html#4) is not set yet
  * These users are banned and won't be assigned:
    * @paulodamaso[/z](https://www.0crat.com/u/paulodamaso): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C3NDPUA8L?q=%7Bjob%3A%27gh%3Azerocracy%2Ffarm%231751%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by g4s8** [comment]
@paulodamaso not only point, you've receive Zerocracy shares, you can check it on project page: https://www.0crat.com/p/C3NDPUA8L. The project doesn't have cash right now, so it's not possible to pay with cash (project can pay cash only for estimated jobs, because it has locked cash for it)
![scrot-2019-01-03-18 59 12](https://user-images.githubusercontent.com/1524988/50647555-ea754180-0f89-11e9-975c-23c244459576.png)

**Comment by paulodamaso** [comment]
@g4s8 So unfortunately I'll never receive this cash or I'll receive it upon recharge?
